src/streaming/controllers/StreamController.js

```diff
@@ -1270,12 +1279,15 @@ function StreamController() {
 
             let allUTCTimingSources = (!adapter.getIsDynamic()) ? manifestUTCTimingSources : manifestUTCTimingSources.concat(customParametersModel.getUTCTimingSources());
 
-            // It is important to filter before initializing the baseUrlController. Otherwise we might end up with wrong references in case we remove AdaptationSets.
-            capabilitiesFilter.filterUnsupportedFeatures(manifest)
-                .then(() => {
-                    baseURLController.initialize(manifest);
-                    timeSyncController.attemptSync(allUTCTimingSources, adapter.getIsDynamic());
-                });
+            // WebRTC TODO: It is important to filter before initializing the baseUrlController. Otherwise we might end up with wrong references in case we remove AdaptationSets.
+            // capabilitiesFilter.filterUnsupportedFeatures(manifest)
```

Comment:
Suggestion: Should we add a new setting to `streaming.capabilities` to deactivate codec filtering? Then we can leave this line untouched.

Reply:
Well optimally I think it might make sense to move the WebRTC logic to its own "module/plugin" ( similar to MssHandler ). But I will admit I haven't looked into how feasible that would be.

In that case the `capabilitiesFilter` ( i'd imagine ) would also check for WebRTC support?

Reply:
That sounds reasonable. In addition, if the WebRTC logic is implemented as a separate module we can consider moving the changes to the main branch. I resolve this for now, but we should not miss this one commented line `capabilitiesFilter.filterUnsupportedFeatures(manifest)` if we do a merge.